src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
@timmywil Do you think the same about comments like "// Support: IE 8-11 only"? We have a couple of those and I didn't remove IE8 mentions.

If we took such a stance to the extreme, we'd have to also change comments like `Chrome 40-48+` to `Chrome 48+` or `Chrome 47-48+` as we don't support older versions of Chrome than the previous one.